samcli/commands/local/invoke/cli.py

```diff
@@ -21,23 +22,27 @@
 LOG = logging.getLogger(__name__)
 
 HELP_TEXT = """
-You can use this command to execute your function in a Lambda-like environment locally.
-You can pass in an event body using the -e (--event) parameter.
-Logs from the Lambda function will be written to stdout.\n
-\b
-Invoking a Lambda function without an input event
-$ sam local invoke "HelloWorldFunction"\n
-\b
-Invoking a Lambda function using an event file
-$ sam local invoke "HelloWorldFunction" -e event.json\n
-\b
-Invoking a Lambda function using input from stdin
-$ echo '{"message": "Hey, are you there?" }' | sam local invoke "HelloWorldFunction" --event - \n
+    Invoke AWS serverless functions locally.
```

Comment:
I know we use serverless functions interchangeably with lambda functions, but I'm wondering if the description here should list both in case customers are confused about it working with `AWS::Severless::..` resources versus non serverless resources.

I might be thinking too much into this and it might be overly nit-picky, especially considering the description below. Feel free to ignore this if you think it's not necessary.